modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -642,6 +644,10 @@ exports.sanitizeProfile = function(profile, authenticatedUser) {
   delete profile.password;
   delete profile.salt;
 
+  // This information is not sensitive, but isn't needed at frontend
+  delete profile.publicReminderCount;
```

Comment:
Should probably change all these deletes into a whitelist instead at some point, so new params aren't automatically sent to frontend...